staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -21,14 +21,13 @@ set -o pipefail
 # generate-internal-groups generates everything for a project with internal types, e.g. an
 # user-provided API server based on k8s.io/apiserver.
 
-if [ "$#" -lt 5 ] || [ "${1}" == "--help" ]; then
+if [ "$#" -lt 4 ] || [ "${1}" == "--help" ]; then
   cat <<EOF
-Usage: $(basename "$0") <generators> <output-package> <internal-apis-package> <extensiona-apis-package> <groups-versions> ...
```

Comment:
is this script still actually doing anything with the internal groups? is the intent to delete this script completely by the time you finish carving up https://github.com/kubernetes/kubernetes/pull/114831?

internal and external packages are not always nested (see k8s.io/api and k8s.io/kubernetes/pkg/apis split), so I'm not sure this is still usable by anyone with that split if we collapse the two args to one

Reply:
This script will go away completely, and possibly be relaced by something nicer when go workspaces lands.  AFAICT we have no such users in-tree, but I don't know if we want to maintain this script as an external-facing API.  It *only* works with GOPATH.

Reply:
Dropping the script completely and replacing with something nicer sounds great... as a data point, there's a *lot* of invocations of this [in the wild](https://github.com/search?type=code&q=%2F%5C%2Fgenerate-internal-groups.sh+%5B%22a-z%5C%5C%24%5D%2F+-org%3Akubernetes+-org%3Akubernetes-sigs).

I'd like @sttts' feedback on dropping the ability to run this with internal/external API packages located in different places. If we decide that's ok to drop, should we keep the two args, check they are identical, and error if they aren't? That would preserve existing invocations for folks who aren't relying on internal/external API packages being in different locations.